content/en/network_monitoring/performance/network_page.md

```diff
@@ -181,6 +181,24 @@ If you have [setup][9] enhanced resolution for AWS or Azure, NPM can filter and
 
  {{< /tabs >}}
 
+#### Azure
+{{< tabs >}}
```

Comment:
@CordeliaLopez are you planning to add more tabs to this section? If not can you replace the tabs with a header instead?

```suggestion
##### Loadbalancers and Application Gateways
```

Reply:
Maybe in the future, but I will change to tabs when we add more features. Thanks!